hack/istio/tempo/minio.yaml

```
@@ -0,0 +1,87 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  # This name uniquely identifies the PVC. Will be used in deployment below.
+  name: minio-pv-claim
+  labels:
+    app: minio-storage-claim
+spec:
+  # Read more about access modes here: http://kubernetes.io/docs/user-guide/persistent-volumes/#access-modes
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    # This is the request for storage. Should be available in the cluster.
+    requests:
+      storage: 5Gi
```

Do we really need 5 GB committed to the cluster? That's alot for a demo/testing. Can we lower it to something even to 1 GB, or preferrably something in the MB range like `256Mi`?
```suggestion
      storage: 1Gi
```

---

I will test a lower value

---

It looks like it works with no problems (With istio and Kiali). I will change it